new_project/examples/separate_apps_testnet/README.md

```diff
@@ -0,0 +1,204 @@
+In a more real world scenario maker and taker would of course be separate people on separate machines.
+
+## Setup
+
+### Run cnd taker
+
+```toml
+[network]
+listen = ["/ip4/0.0.0.0/tcp/9939"]
+
+[bitcoin]
+network = "testnet"
+node_url = "http://bobtimus.coblox.tech:18332"
+
+[ethereum]
+node_url = "http://bobtimus.coblox.tech:8545"
+
+[http_api]
+address = "0.0.0.0"
+port = 8000
+```
+
+Example seed file for the taker:
+```
+-----BEGIN SEED-----
+ZaBZdOwqLWHTLe9HXEc//wVGdt4VzVkDB/jB7tzkf/c=
+-----END SEED-----
+```
+
+Startup command:
+```
+cnd --config taker_config.toml --seed-file taker_seed.pem
+```
+
+### Run cnd maker
+
+```toml
+[network]
+listen = ["/ip4/0.0.0.0/tcp/19939"]
+
+[bitcoin]
+network = "testnet"
+node_url = "http://bobtimus.coblox.tech:18332"
+
+[ethereum]
+node_url = "http://bobtimus.coblox.tech:8545"
+
+[http_api]
+address = "0.0.0.0"
+port = 18000
+```
+
+Example seed file for the maker:
+```
+-----BEGIN SEED-----
+AaBZdOwqLWHTLe9HXEc//wVGdt4VzVkDB/jB7tzkf/c=
+-----END SEED-----
+```
+
+Startup command:
+```
+cnd --config maker_config.toml --seed-file maker_seed.pem
+```
+
+### Key/account generation and funding for both wallets
+
+Keep in mind that when running both maker and taker on the same machine you have to create a Bitcoin and Ethereum account for both maker and taker, thus you have to create and fund **two Bitcoin** and **two Ethereum** accounts.
+
+Configuration in `env` file (assuming that you run on both maker and taker from the same machine):
+
+```
+BITCOIN_HD_KEY_TAKER=EXTENDED_PRIVATE_KEY_GENERATED_BY_WAGYU
+ETHEREUM_KEY_TAKER=ETHEREUM_PRIVATE_KEY_GENERATED_BY_WAGYU
+
+BITCOIN_HD_KEY_MAKER=EXTENDED_PRIVATE_KEY_GENERATED_BY_WAGYU
+ETHEREUM_KEY_MAKER=ETHEREUM_PRIVATE_KEY_GENERATED_BY_WAGYU
+
+HTTP_URL_CND_TAKER=http://localhost:8000
+HTTP_URL_CND_MAKER=http://localhost:18000
+
+BITCOIN_P2P_URI=bobtimus.coblox.tech:18332
+ETHEREUM_NODE_HTTP_URL=https://bobtimus.coblox.tech:8545
+```
+
+#### Bitcoin
+
+1. Generate HD key (master-key) with [Wagyu](https://github.com/ArgusHQ/wagyu)
+    1. `wagyu bitcoin hd --network testnet --derivation "m"`
+2. Copy the `Extended Private Key` generated by Wagyu into the `env` file.
+3. Generate an address to fund with [Wagyu](https://github.com/ArgusHQ/wagyu)
+    1. `wagyu bitcoin import-hd --derivation "m/44'/1'/0'/0/0" --extended-private GENERATED-EXTENDED-PRIVATE-KEY`
+    2. Fund the `Address` returned
+4. Fund the address through a [Bitcoin testnet coin faucet](https://coinfaucet.eu/en/btc-testnet/)
+
+#### Ethereum
+
+1. Generate an Ethereum account with [Wagyu](https://github.com/ArgusHQ/wagyu)
+    1. `wagyu ethereum hd`
+2. Copy the `Private Key` (NOT the extended private key) to the env file.
+3. Fund the account through an [Ethereum ropsten coin faucet](https://faucet.metamask.io/)
+
+Note: Since Ethereum is account based there is no need to use the extended private key to derive multiple accounts.
```

Comment:
This sentence does not make much sense 🙃 In theory we would not need an HD key on bitcoin neither. We could just reuse the same key over and over again. Instead of explaining why one should not do that on bitcoin I would just remove that sentence.
```suggestion
```